static/app/components/organizations/pageFilters/container.tsx

```diff
@@ -91,7 +91,9 @@ function Container({
     ? specifiedProjects
     : specifiedProjects.filter(project => project.isMember);
 
-  const doInitialization = () =>
+  const isStarfishPage = router?.location?.pathname?.startsWith('/starfish');
```

Comment:
Can this be a prop? I really really want to exclude explicit knowledge of Starfish from outside components

Reply:
Fair ask! I'll change it so it's a prop